test/src/journey.test.js

```diff
@@ -0,0 +1,57 @@
+const modulePath = 'src/journey';
```

Comment:
Should this journey test include and `Exit` step?

Reply:
👍